tt1-blocks/style.css

```diff
@@ -4,10 +4,10 @@ Theme URI: https://github.com/WordPress/theme-experiments/tree/master/tt1-blocks
 Author: the WordPress team
 Author URI: https://wordpress.org/
 Description: TT1 Blocks is an experimental block-based version of the Twenty Twenty-One theme. It is built to leverage the full-site editing functionality that is being built in the Gutenberg plugin. This theme is not meant for use on a production site.
-Requires at least: 5.6
-Tested up to: 5.8
+Requires at least: 6.0
+Tested up to: 6.0
 Requires PHP: 5.6
-Version: 0.4.8
+Version: 0.4.9
```

Comment:
do you think these changes justify the bump to the 1.0.0 version? I don't think we should consider tt1blocks experimental anymore if tt2 isn't either

Reply:
Yeah we could do that,